### PR TITLE
Some optimization

### PR DESCRIPTION
**Pull request by itmard**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -112,7 +112,7 @@
                             {% include 'font_why.html' %}
                         </div>
                         <div class="tab-pane fade svn text-center" id="svn">
-                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}">
+                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}" width="362" height="362">
```
Comment (hamidfzm) [inline templates/index.html]: width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard

File: templates/index.html [hunk 2]
```diff
@@ -126,7 +126,7 @@
                 <div class="panel-body">
                     <div class="tab-content">
                         <div class="tab-pane fade in active home justify" id="home">
-                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}">
+                            <img class="center-block" src="{{ url_for('static', filename='image/why.png') }}" width="362" height="362">
```
Comment (hamidfzm) [inline templates/index.html]: width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard

File: templates/index.html [hunk 3]
```diff
@@ -199,7 +199,7 @@ <h1 class="top-bottom-padding">حمایت‌کنندگان</h1>
 
         <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 col-sm-12">
             {% for donator in donatores %}
-                <img src="{{ donator.avatar }}" class="avatar" title="{{ donator.name }}">
+                <img src="{{ donator.avatar }}" class="avatar" width="64" height="64" title="{{ donator.name }}">
```
Comment (hamidfzm) [inline templates/index.html]: width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard

File: templates/index.html [hunk 4]
```diff
@@ -59,16 +59,16 @@
                 <!-- Wrapper for slides -->
                 <div class="carousel-inner">
                     <div class="item active">
-                        <img src="{{ url_for('static', filename='image/carousel1.jpg') }}">
+                        <img src="{{ url_for('static', filename='image/carousel1.jpg') }}" width="1920" height="700">
```
Comment (hamidfzm) [inline templates/index.html]: width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard

**Comment by itmard** [comment]
We need to do this for optimization

File: templates/index.html (hunk 1 above)
Comment (itmard) [inline templates/index.html]: We need to do this for optimization

File: templates/index.html (hunk 2 above)
Comment (itmard) [inline templates/index.html]: We need to do this for optimization

File: templates/index.html (hunk 3 above)
Comment (itmard) [inline templates/index.html]: We need to do this for optimization

Comment (hamidfzm) [inline templates/index.html]: It doesn't optimize pictures, it just breaks responsive design.

File: templates/index.html (hunk 2 above)
Comment (hamidfzm) [inline templates/index.html]: It doesn't optimize pictures, it just breaks responsive design

**Pull request merged**